components/decimal/src/provider.rs

```diff
@@ -117,13 +94,13 @@ pub struct GroupingSizesV1 {
 )]
 #[cfg_attr(feature = "serde", derive(serde::Deserialize))]
 pub struct DecimalSymbolsV1<'data> {
-    /// Prefix and suffix to apply when a negative sign is needed.
+    /// Pattern to apply when a negative sign is needed.
     #[cfg_attr(feature = "serde", serde(borrow))]
-    pub minus_sign_affixes: AffixesV1<'data>,
+    pub minus_sign_pattern: Option<SinglePlaceholderPattern<Cow<'data, str>>>,
```

Comment:
Question: why did you make it an `Option`? The value is only 2 bytes long. Seems like unnecessary indirection? You already have a `Cow` so the niche is already being used.